internal/core/src/index/ScalarIndex.h

```diff
@@ -32,11 +33,14 @@ class ScalarIndex : public IndexBase {
     NotIn(size_t n, const T* values) = 0;
 
     virtual const TargetBitmapPtr
-    Range(T value, OperatorType op) = 0;
+    Range(T value, OpType op) = 0;
 
     virtual const TargetBitmapPtr
     Range(T lower_bound_value, bool lb_inclusive, T upper_bound_value, bool ub_inclusive) = 0;
 
+    virtual T
+    Reverse_Lookup(size_t offset) const = 0;
```

Comment:
I was wondering that should all index implement this method? Especially in our situation, if the raw data of a field was loaded, it's not necessary and a big waste to do this.